src/test/java/org/apereo/openequella/tools/toolbox/TestUtils.java

```diff
@@ -0,0 +1,114 @@
+		Config.getInstance().setConfig(Config.CF_DB_PASSWORD, envSpecific.getProperty(Config.CF_DB_PASSWORD));
+		Config.getInstance().setConfig(Config.CF_FILESTORE_DIR, envSpecific.getProperty(Config.CF_FILESTORE_DIR));
+		Config.getInstance().setConfig(Config.CF_EMAIL_RECIPIENTS, envSpecific.getProperty(Config.CF_EMAIL_RECIPIENTS));
+		Config.getInstance().setConfig(Config.CF_NUM_OF_ITEMS_PER_QUERY, "20");
+		final String advFsKeyBeta = "cf.filestore.advanced.812.44ac1c1a-dd0c-462f-b717-53324c0dd6f9";
+		Config.getInstance().setConfig(advFsKeyBeta, envSpecific.getProperty(advFsKeyBeta));
+		final String advFsKeyBetaInst2 = "cf.filestore.advanced.1509.44ac1c1a-dd0c-462f-b717-53324c0dd6f9";
+		Config.getInstance().setConfig(advFsKeyBetaInst2, envSpecific.getProperty(advFsKeyBetaInst2));
+		final String advFsKeyCharlie = "cf.filestore.advanced.812.5a7b2083-2671-414b-8e5b-c4cd9dfa1f30";
+		Config.getInstance().setConfig(advFsKeyCharlie, envSpecific.getProperty(advFsKeyCharlie));
+		Config.getInstance().setConfig("cf.db.filestore.institution.handle.checkFilesTesting2", "checkFilesTesting2Renamed");
+	}
+
+	public static void buildEmailProps() {
+		Properties envSpecific = loadEnvSpecificDefaults();
+		Config.getInstance().setConfig(Config.EMAIL_SERVER, "smtp.gmail.com:587");
+		Config.getInstance().setConfig(Config.EMAIL_SENDER_NAME,
+						"testEmailSettingsE2E");
+		Config.getInstance().setConfig(Config.EMAIL_SENDER_ADDRESS,
+						"testEmailSettingsE2E@example.com");
+		Config.getInstance().setConfig(Config.EMAIL_USERNAME,
+						envSpecific.getProperty(Config.EMAIL_USERNAME));
+		Config.getInstance().setConfig(Config.EMAIL_PASSWORD,
+						envSpecific.getProperty(Config.EMAIL_PASSWORD));
+	}
+
+	public static Properties loadEnvSpecificDefaults() {
+		final String path = "conf/junit.properties";
+		try (InputStream input = new FileInputStream(path)) {
+			LOGGER.debug("Using [{}] as the configuration file for env specific junit properties.", path);
+			Properties envSpecificDefaults = new Properties();
+			envSpecificDefaults.load(input);
+			return envSpecificDefaults;
+		} catch (FileNotFoundException e) {
+			LOGGER.warn("Unable to find the config file: {}", e.getMessage());
+		} catch (IOException e) {
+			LOGGER.warn("Unable to use the config file: [{}] due to - [{}].", path, e.getMessage());
+		}
+		return null;
+	}
+
+	public static void debugDumpList(List<String> strs) {
+		if(LOGGER.isDebugEnabled()) {
+			LOGGER.debug("Begin Result Dump");
+			for(int i = 0; i < strs.size(); i++) {
+				LOGGER.debug("[{}] - {}", i, strs.get(i));
+			}
+			LOGGER.debug("End Result Dump");
+		}
+	}
+
+//	public static void resetTestOutputDirectory() {
+//		File dir = new File(Config.get(Config.CF_OUTPUT_FOLDER));
+//		String[] errorStats = dir.list(new FilenameFilter() {
+//			@Override
+//			public boolean accept(File file, String filename) {
+//				return !filename.equals("placeholder");
+//			}
+//		});
+//
+//		for(String f : errorStats) {
```

Comment:
Is this intended to be commented out?

Reply:
thanks for the catch.  Should have been removed.